rbc/tests/test_externals_libdevice.py

```diff
@@ -84,4 +88,7 @@ def test_externals_libdevice(heavydb, fname, retty, argtys, has_ptr_arg):
         cols = ", ".join(tuple(map(lambda x: cols_dict[x], argtys)))
         query = f"SELECT {func_name}({cols}) FROM {table}"
 
-    _, _ = heavydb.sql_execute(query)
+    _, result = heavydb.sql_execute(query)
+
+    assert fname in str(fns[fname])
```

Comment:
This may give false-positive result. In fact, locally, I get
```
$ pytest -sv rbc/tests/test_externals_libdevice.py -x
<snip>
>       assert fname in str(fns[fname])
E       assert '__nv_abs' in "test_externals_libdevice_abs['int32(int32), device=gpu']"
E        +  where "test_externals_libdevice_abs['int32(int32), device=gpu']" = str(Caller(<function define.<locals>.inner.<locals>.fn at 0x7fd3dfbc4310>, Signature('int32(int32), device=gpu')))
```